account_invoice_line_sequence/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Provides a new sequence field on invoice lines which helps to manage the order of the invoice lines.
```

Comment:
Can you improve the readme to mention that the sequence changes automatically when editing the lines and the sequence number is printed in the report?